packages/react-components/src/components/styles/StreamMedia.styles.ts

```diff
@@ -21,7 +21,18 @@ export const mediaContainer = (theme: Theme): string =>
 /**
  * @private
  */
-export const invertedVideoStyle = (theme: Theme): string =>
+export const invertedVideoInPipStyle = (theme: Theme): string =>
   mergeStyles(mediaContainer(theme), {
-    transform: 'rotateY(180deg)'
+    // The HTMLElement returned by the headless SDK is already inverted.
+    // But in picture-in-picture mode, we do not want to invert the host HTMLElement.
+    // Instead, we need to target the :picture-in-picture pseudoclass.
+    //
+    // First reset the host HTMLElement.
+    transform: 'rotateY(180deg)',
```

Comment:
Today we apply `transform: 'rotateY(180deg)'` - is this un-rotating it? - Or how does this 'reset' the host element?

Reply:
Ooo I see we weren't passing in `isMirrored=true` to StreamMedia lol so this was never being applied, instead of flipping another element, can we set `transform: 'rotateY(0deg)'` to whatever the CallingSDK is flipping? (though maybe a hacky div selector might be even more reliant on the calling sdk render element so would be a bad idea)

Reply:
Yeah I decided double flipping is better than poking into whatever headless returns.

For example, I didn't know that they return a the `<video>` inside a container `<div>`. It's the `<div>` that is transformed, but it could also be the `<video>` itself.